pkg/pillar/hypervisor/containerd.go

```diff
@@ -26,7 +26,7 @@ type ctrdContext struct {
 }
 
 func initContainerd() (*ctrdContext, error) {
-	ctrdClient, err := containerd.NewContainerdClient()
+	ctrdClient, err := containerd.NewContainerdClient(false)
```

Comment:
Same for here. Should this not be in the _user_ containerd, i.e. the only one that really exists as far as pillar is concerned?